package.js

```diff
@@ -0,0 +1,19 @@
+var packageName = 'rubenv:angular-gettext';
+
+var where = 'client';
+var version = '2.1.0';
```

Comment:
Couldn't do it exactly like that due to the nature of package.js not running in a "common" node.js environment (`require` not available, can't read with `fs.readFileSync` either because no `__dirname` available).

But since grunt-bump just looks for something like `"version": "2.1.0"` anyway, just creating an options object works:

```javascript
var options = {
    "version": "2.1.0",
    "where": "client",
    "packageName": "rubenv:angular-gettext"
};
```